[NVSHAS-5843] Adding Documentation for Federated ConfigMap YAML

sunilarjun opened the pull request:
Adding documentation for the Federated ConfigMap primary and managed clusters. Also updated the latest ConfigMap link to https://raw.githubusercontent.com/neuvector/manifests/main/kubernetes/5.4.0/initcfg.yaml, and updated the sample `initcfg.yaml`. Related to NVSHAS-5843.

williamlin-suse commented on docs/02.deploying/01.production/01.configmap/01.configmap.md:
```diff
@@ -13,13 +13,46 @@ The 'always_reload: true' setting can be added in any ConfigMap yaml to force re
+    Join_Token: 2be93d8f-d42a-44fc-9d33-7c1a6153066b
+    # Required: REST server/port of the neuvector-svc-controller-fed-master service
+    Primary_Rest_Info:
+      Server: 4.3.2.1
```
This line is misleading. 
1.2.3.4 is used as the primary cluster's IP in above sample.
It's better use use 1.2.3.4 for Primary_Rest_Info.Server in this line

sunilarjun replied:
Updated in https://github.com/neuvector/docs/commit/150fcfa6f883e013091dc3098f80d9f5c261c08a

williamlin-suse commented on docs/02.deploying/01.production/01.configmap/01.configmap.md:
```diff
@@ -13,13 +13,46 @@ The 'always_reload: true' setting can be added in any ConfigMap yaml to force re
+      Port: 11443
+    # Optional, for remote cluster only. REST server/port of the neuvector-svc-controller-fed-managed service
+    Managed_Rest_Info:
+      Server: 1.2.3.4
```
It's better use use 4.3.2.1 for Managed_Rest_Info.Server in this line

williamlin-suse commented on docs/02.deploying/01.production/01.configmap/01.configmap.md:
```diff
@@ -13,13 +13,46 @@ The 'always_reload: true' setting can be added in any ConfigMap yaml to force re
+    # Optional. If specified, it overwrites the cluster name specified in the system configuration in the sysinitcfg.yaml file
+    Cluster_Name: remote.cluster.local
+    # Required and must be the same on primary cluster and remote clusters
+    # It must be 36 characters long, i.e., 32 hex characters grouped as 8-4-4-4-12 and separated by four hyphens
```
Please add a note: the Join_Token specified in the remote cluster's fedinitcfg.yaml needs to be the same as the Join_Token specified in the primary cluster's fedinitcfg.yaml
Otherwise the auto-joining request will be declined by the primary cluster.

sunilarjun replied:
Updated in https://github.com/neuvector/docs/commit/150fcfa6f883e013091dc3098f80d9f5c261c08a

williamlin-suse commented:
Who will update https://open-docs.neuvector.com/ about the changes in the PR?

sunilarjun commented:
> Who will update https://open-docs.neuvector.com/ about the changes in the PR?

Thank you for reviewing! I've addressed the feedback in this commit: https://github.com/neuvector/docs/pull/134/commits/150fcfa6f883e013091dc3098f80d9f5c261c08a

For clarification this PR is handling the content update for the documentation at https://open-docs.neuvector.com/, and I will be handling the port of this PR to the product docs (https://documentation.suse.com/cloudnative/security/5.4/en/overview.html) as well once approved.
